app/src/NativeBase/Components/StaRibbon.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file for StaRibbon component
+ */
+import { HStack, Box } from 'native-base'
+import React from 'react'
+
+const StaRibbon = () => {
```

Comment:
Optional little bit of shorthand to make this file simpler, because this component just returns JSX without any other logic:
you can make this line `const StaRibbon = () => (`, remove the following line, and remove the closing curly bracket on the penultimate line of the file.

Reply:
I have now removed the unnecessary curly braces from the sta ribbon.